apps/wing-console/console/app/src/index.ts

```diff
@@ -65,8 +65,8 @@ export const createConsoleApp = async (options: CreateConsoleAppOptions) => {
     analyticsAnonymousId: analyticsStorage.getAnonymousId(),
     analytics,
     async requireSignIn() {
-      if (options.requireSignIn === false) {
-        return false;
+      if (options.requireSignIn) {
+        return options.requireSignIn;
       }
```

Comment:
This is removing a very important behavior: skipping the sign in prompt when the env variable `NO_SIGN_IN` is `true`.

Reply:
isn't this `requireSignIn` flag used to force the modal behaviour? I think it is not passed by default so it will go through the default behavior.

Reply:
Ahh yes, but `if (options.requireSignIn !== undefined)`

Reply:
this should be 
```
if (options.requireSignIn !== undefined) {
  return options.requireSignIn;
}
```